examples/modal.py

```diff
@@ -0,0 +1,67 @@
+import discord
+from discord import app_commands, Object, ui
+
+# Just default intents and a `discord.Client` instance
+# We don't need a `commands.Bot` instance because we are not
+# creating text-based commands.
+intents = discord.Intents().default()
+client = discord.Client(intents=intents)
+
+# We need an `discord.app_commands.CommandTree` instance to register slash commands
+tree = app_commands.CommandTree(client)
+
+# The guild in which this slash command will be registered.
+# As global commands can take up to an hour to propagate, it is ideal
+# to test it in a guild.
```

Comment:
Generally, I use the term "application commands" instead of "slash commands" but I guess for examples this is fine.